crates/matrix-sdk/src/send_queue.rs

```diff
@@ -601,7 +604,7 @@ fn convert_error_to_wedged_reason(value: &crate::Error) -> QueueWedgeError {
             }
             SessionRecipientCollectionError::VerifiedUserChangedIdentity(users) => {
                 QueueWedgeError::IdentityViolations {
-                    users: users.iter().map(OwnedUserId::to_string).collect(),
+                    users: users.iter().map(ruma::OwnedUserId::to_string).collect(),
```

Comment:
nit: unnecessary qualification here

Reply:
Outdated now, but it was previously because of the `#[cfg(feature = "e2e-encryption")]`. Other option could be to add the feature requirement at the import level (if not would be unused import)